clients/lsp-clangd.el

```diff
@@ -47,6 +47,35 @@
 (declare-function flycheck-error-group "ext:flycheck" (err) t)
 (declare-function flycheck-error-message "ext:flycheck" (err) t)
 
+(defcustom lsp-clangd-download-url
+  (pcase system-type
+    ('gnu/linux "https://github.com/clangd/clangd/releases/download/12.0.0/clangd-linux-12.0.0.zip")
```

Comment:
Maybe have a different defcustom for the version? to avoid users changing both here and the `lsp-clangd-binary-path` too

Reply:
On the VS Code side they parse the latest version from https://api.github.com/repos/clangd/clangd/releases/latest, but I don't know if it's worth the complexity; we could simply add a variable for the version.

Reply:
> On the VS Code side they parse the latest version from https://api.github.com/repos/clangd/clangd/releases/latest, but I don't know if it's worth the complexity; we could simply add a variable for the version.

Yes. Given the 6months release cycle, it does not pay the effort.